### DISCUSSION TITLE
using MXNET with c++

### DISCUSSION THREAD

**Discussion by jbensabat** [apache/incubator-mxnet#19857] [General] [answered]
Hello 
I have downloaded mxnet and built the lib under windows 10/64 and MSVC 2019
I am now trying to find the cpp examples described in the tutorial but they are missing 
I saw that this directory exists in version 1.6.4 but some of the functions are missing so the examples do not compile.
I am just trying to use the code with the mlp_cpu.cpp example 
where can I find this file which compiles with 1.7.0 ?
thanks
jac

**Comment by szha** [comment, marked as answer]
C++ API and examples for 1.7.0 are in https://github.com/apache/incubator-mxnet/tree/v1.x/cpp-package
